ethereum/spec/src/main/java/tech/pegasys/teku/spec/SpecFactory.java

```diff
@@ -27,17 +31,30 @@ public static Spec createMinimal() {
     return create(Eth2Network.MINIMAL);
   }
 
+  public static Spec createMinimalAltair() {
+    final SpecConfigAltair specConfig =
+        SpecConfigAltair.required(SpecConfigLoader.loadConfig(Eth2Network.MINIMAL.configName()));
+    return create(specConfig, specConfig.getAltairForkVersion());
+  }
+
   public static Spec createMainnet() {
     return create(Eth2Network.MAINNET);
   }
 
   public static Spec create(final String configName) {
```

Comment:
Maybe we should rename this:
```suggestion
  public static Spec createPhase0(final String configName) {
```

Reply:
I've moved the `createMinimalAltair` out to a test class so will leave this as is. There's some fun to be had in the future when we inevitably launch a testnet that skips phase0 and starts with whatever the latest fork is at the time. I don't see a way to specify that with the current config setup, but in theory this `create` won't always create a phase0 spec version.

In fact even when Altair is actually scheduled it will have to be modified to create a `ForkManifest` that has both phase0 and Altair included.  I think eventually that will come from a `ALTAIR_FORK_SLOT` config item but it doesn't exist yet...

Reply:
Right - there's some work to do on setting up forks correctly.  Looks like we'll probably have to scan through the config and pull out relevant fork / slot fields if they exist.  The current altair config in the specs repo actually has an altair slot defined, but it's set to 0.  So, it seemed safer to leave that out for now 😬